cmd/lakectl/cmd/branch_tools_test.go

```diff
@@ -0,0 +1,105 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/treeverse/lakefs/pkg/metastore"
+	"github.com/treeverse/lakefs/pkg/metastore/mock"
+)
+
+const (
+	dbName     = "source"
+	repoName   = "repo"
+	branchName = "branch"
+)
+
+var (
+	validPath   = fmt.Sprintf("s3://%s/%s/path/to/schema", repoName, branchName)
+	invalidPath = fmt.Sprintf("s3/%s/%s/", repoName, branchName)
+)
+
+func TestExtractRepoAndBranchFromDBName(t *testing.T) {
+	type args struct {
+		ctx    context.Context
```

Comment:
If we never pass a context, can we just have `dbName` here?  (If we _do_ pass a context, Go generally prefers `context.Background()` or `context.TODO()` to the `nil` context...).